### PR TITLE
add dataset-k8s-instruct

### PR DESCRIPTION
**Pull request by samos123**

### CODE REVIEW

File: dataset-k8s-instructions/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+```sh
+docker build -t dataset-k8s-instructions .
+```
+
+Explore and develop with a Jupyter Lab.
+
+```sh
+# Run a Jupyter Notebook.
+docker run -it -e LOAD_DATA_PATH=/data/all.jsonl -v $(pwd)/data:/data -v $(pwd)/src:/dataset/src -p 8888:8888 dataset-k8s-instructions notebook.sh
```
Comment (nstogner) [inline dataset-k8s-instructions/README.md:17]: Is LOAD_DATA_PATH used?

File: dataset-k8s-instructions/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Dataset: K8s Instructions
```
Comment (nstogner) [inline dataset-k8s-instructions/README.md:1]: I think this can become a simple `dataset-http` data loader that accepts a `url` param.

Comment (nstogner) [reply dataset-k8s-instructions/README.md:1]: (Could be another PR)

**Review by nstogner — APPROVED** [review]
Approved, but check comments

File: dataset-k8s-instructions/README.md (hunk 2 above)
Comment (samos123) [reply dataset-k8s-instructions/README.md:1]: https://github.com/substratusai/images/issues/5

Comment (samos123) [reply dataset-k8s-instructions/README.md:1]: will fix in separate PR

File: dataset-k8s-instructions/README.md (hunk 1 above)
Comment (samos123) [reply dataset-k8s-instructions/README.md:17]: nope, let me fix the readme.

**Pull request merged**